services/core-api/app/api/services/document_manager_service.py

```diff
@@ -79,7 +89,17 @@ def initializeFileUploadWithDocumentManager(cls, request, mine, document_categor
             data=data,
             cookies=request.cookies)
 
-        return Response(str(resp.json()), resp.status_code, resp.raw.headers.items())
+        resp = Response(str(resp.json()), resp.status_code, resp.raw.headers.items())
+
+        if Config.ENVIRONMENT_NAME != 'prod':
+            project = Project.find_by_project_guid(project_guid)
+
+            if resp:
+                renotify_hours = 24
+                trigger_notification(f'File(s) in project {project.project_title} has been updated for mine {mine.mine_name}.',
```

Comment:
I think this is likely not the right place to put this as it's called when we first initialize the file upload. We don't want to send it until the file Upload has been finished and the related MineDocument has been created. I think you'll need to put this where the `MineDocument` for all the different categories (major mine application / project summary / project decision package / information requirements table)

Reply:
Updated accordingly.